modules/programs/ssh.nix

```diff
@@ -144,6 +152,7 @@ let
     ++ optional cf.identitiesOnly         "  IdentitiesOnly yes"
     ++ optional (cf.user != null)         "  User ${cf.user}"
     ++ optional (cf.identityFile != null) "  IdentityFile ${cf.identityFile}"
+    ++ optional (cf.certificateFile != null) "  CertificateFile ${cf.certificateFile}"
```

Comment:
can you indent the rest of the line on the quote similar to the rest of the lines?